app/javascript/packs/date.js

```diff
@@ -0,0 +1,7 @@
+$(function () {
+  $('#to_date_timepicker').datetimepicker({ format: "YYYY/MM/DD" });
```

Comment:
Add a `datetimepicker-input` class to the element so we don't need to rewrite same code.
```suggestion
  $('.datetimepicker-input').datetimepicker({ format: "YYYY/MM/DD" });
```